conf/libvirt.yaml.template

```diff
@@ -0,0 +1,12 @@
+COMPUTE_RESOURCES:
+  # External Libvirt Hostname
+  LIBVIRT_HOSTNAME: libvirt.example.com
```

Comment:
if these values are going to get updated based on git lab PR, we should comment those attributes